Helm warning when setting some tolerations in values.yaml in crowdsec Helm chart

When setting some tolerations on agents, in values.yaml, helm displays the following warning:
> coalesce.go:286: warning: cannot overwrite table with non table for crowdsec.agent.tolerations (map[])

I suppose it's because it is set as an object in bundled values.yaml with `{}`, and overridden as a map in our values.yaml

--- a/charts/crowdsec/values.yaml
+++ b/charts/crowdsec/values.yaml
@@ -197,7 +197,7 @@ lapi:
   # -- nodeSelector for lapi
   nodeSelector: {}
   # -- tolerations for lapi
-  tolerations: {}
+  tolerations: []
 
   # -- Enable service monitoring (exposes "metrics" port "6060" for Prometheus)
   metrics:
@@ -284,7 +284,7 @@ agent:
   # -- nodeSelector for agent
   nodeSelector: {}
   # -- tolerations for agent
-  tolerations: {}
+  tolerations: []
 
   # -- Enable service monitoring (exposes "metrics" port "6060" for Prometheus)
   metrics:

